zip-0320.rst

```diff
@@ -219,20 +221,25 @@ executing the following steps:
    two-byte address prefix is not :math:`[\mathtt{0x1C}, \mathtt{0xB8}]`,
    return an error. Otherwise, let the **validating key hash** be the remaining
    20 bytes of the array after removing the two-byte address prefix.
-3. Construct a new Unified Address using a single Traceable P2PKH Receiver
-   :math:`\mathtt{0x04}` with the 20-byte **validating key hash** as its value.
-   In addition, metadata items such as an Address Expiry Height or Address
-   Expiry Date [#zip-0316-address-expiry]_ MAY be included.
+3. Construct a new Revision 1 Unified Address using a single P2PKH Receiver
+   :math:`\mathtt{0x04}` with the 20-byte **validating key hash** as its value,
+   and a Source Restriction Metadata Item (Typecode :math:`\mathtt{0xE2}`)
+   having value :math:`\mathtt{0x00}` (Transparent Source Only). In addition,
+   metadata items such as an Address Expiry Height or Address Expiry Date
+   [#zip-0316-address-expiry]_ MAY be included.
+4. Encode the Unified Address using the “``u+``” Human Readable Part as
```

Comment:
Noting that `u+` is a placeholder here, until we have decided on what HRPs to use.

Reply:
:heavy_check_mark: